src/pathfinding_service/constants.py

```diff
@@ -26,3 +27,37 @@
 
 MAX_AGE_OF_IOU_REQUESTS: timedelta = timedelta(hours=1)
 MAX_AGE_OF_FEEDBACK_REQUESTS: timedelta = timedelta(minutes=10)
+
+DISCLAIMER: str = textwrap.dedent(
```

Comment:
This still exists twice

Reply:
yes, I am unsure if we get 2 or just 1 disclaimer from Jim. I will adjust accordingly